addons/rating/views/rating_rating_views.xml

```diff
@@ -68,7 +68,7 @@
                                     </div>
                                     <div class="col-8 pl-1">
                                         <strong>
-                                            <field name="rated_partner_id" />
+                                            <field name="rated_partner_name"/>
```

Comment:
@xavierbol , Here PO want the only name of the rated operator, not his company(and as per your suggestion it will get the display name of partner with company). so either we have to add the related field of name  OR in main base/res_partner.py support context in name_get to display only name of the partner without company.

Let me know if you have better suggestion for this? Or should we convince the PO to display the name of company too?

Thanks

Reply:
@xavierbol , Here we wants the value of only name(not with company) and the field is directly loaded using the `search_read` in the kanban so it will not call the `name_get`. so we can not use the context key. 

so one ways is to add the related field display it `OR` split the value by comma(,) in xml and display the name but that's not preferable.

what do you think?

Thanks

Reply:
I think you can leave it like you did. 🙂 
Thanks for the explanation.